python/feathub/examples/streaming_average_flink_cli.py

```diff
@@ -84,7 +84,7 @@ def main() -> None:
     )
 
     sink = PrintSink()
-    client.materialize_features(feature_view, sink=sink, allow_overwrite=True).wait()
+    client.materialize_features(feature_view, sink=sink, allow_overwrite=True)
```

Comment:
If we remove this `wait()`, can we still ensure that the `PrintSink()` will print all results before the program exits?